data_platform_with_deep_indexed_data_and_staking_reports.md

```diff
@@ -0,0 +1,248 @@
+
+The solution comes with the second layer of data which is data marts layer.
+
+There are two ways to create report based on raw layer:
+
+- via BI where the query will be store as request to DWH
+- PR with data mart that should be re-calculated every time
+
+Second option is for popular/the most important reports which are accepted by admin users (p2p.org DWH team).
+
+Initially the solution will contain reports about staking in Polkadot/Kusama/Moonbeam/Moonriver:
+
+- active set of validators per epoch
+- stake amount between nominator and validator for every epoch
+- total validator stake per epoch
+- the validator commission for all active validators
+- the public identity of validator
+- the rewards (tokens, rewards points) earned by nominator with particular validator per epoch
+- total rewards (tokens, rewards points) earned by nominator per epoch
+- total rewards (tokens, rewards points) earned by validator per epoch
+- ability to calculate APR/APY on top of this reports
+
+**Data quality**
+
+Polkadot SLI (based on history for internal consumption):
+
+- Block processing mean time: 1073 msec (min - 238 msec, max- 35 sec)
+- Mean latency between block in RPC node to block in database: ~ 18 сек (min - 13 sec, max 99 sec)
+- Blocks processed correctly in first attempt (others are fixed automatically): ~ 99%
+- Staking SLI: 100% (no problems detected)
+
+Kusama and Moonbeam are mostly similar.
+
+Moonriver sli:
+
+- Block processing mean time: 914 msec (min - 81 msec, max- 98 sec)
+- Mean latency between block in RPC node to block in database: ~ 95 сек (min - 12 sec, max 240 sec)
+- Blocks processed correctly in first attempt (others are fixed automatically) ~ 96%
+- Staking SLI: 95%
+
+We also have an SLI [dashboard here](https://reports.p2p.org/superset/dashboard/124/?native_filters_key=hx3xwFAW7LaoWw6vWbB6eY7JD3BOjObbcgnwy_BkExJw20nkHu2NtfTYLkDe22Kl), but thresholds used in this dashboard were set by internal team and could be different for final public solution.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Developers:
+    - Ivan Safronov (developer of Mbelt)
+    - Artem Kiselev (team lead of indexation team in P2P.org)
+- DWH engineers:
+    - Kovalev German - (data engineer)
+    - Kamalov Sergey (team lead of DWH team in P2P.org)
+- Product owner: Kotenko Nick
+
+### Team Code Repos
+
+Indexer: [Mbelt](https://github.com/p2p-org/polkadot-profit-transformer/tree/mbelt3/prod)
+
+Public DWH: “p2p-data-platform” in bigQuery you can find in public access though bigQ console
+
+### Team's experience
+
+We believe that proposed product can brought not only new user experience for validators and NO managers but provide truly decentralization with the performance improvement. P2P.org has long-term experience in more then 25 blockchains since 2018. We can bring the high level expertise to help new participants perform better by providing recommendations on-top of the core solution.
```

Comment:
```suggestion
We believe that proposed product can bring not only a new user experience for validators and NO managers but also provide true decentralization with the performance improvement. P2P.org has long-term experience in more then 25 blockchains since 2018. We can bring the high level expertise to help new participants perform better by providing recommendations on-top of the core solution.
```